tests/stream_test.c

```diff
@@ -14,6 +14,10 @@
 
 AWS_STATIC_STRING_FROM_LITERAL(s_simple_test, "SimpleTest");
 
+/* 0x1A represents the Windows end-of-file character. Having this in the test data set allows us to verify that file
+ * stream reads on binary files do not terminate early on Windows.*/
+const uint8_t s_simple_binary_test[] = "abcdef\x1Aghijk";
```

Comment:
Defining this with quotes means you get a null terminator at the end of your byte array, did you want a null terminator? If not, consider defining it like:

const uint8_t s_simple_binary_test[] = {'a', 'b', 'c', 'd', 'e', 'f', 0x1A, 'g', 'h', 'i', 'j', 'k'};

Reply:
That's a good point, it's okay for the test, but I'll switch it to an array to be more correct.